src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/ElmWindow.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ElmSharp;
+using System;
+using Tizen.Applications.ComponentBased.Common;
+
+namespace Tizen.Applications.ComponentBased.Default
+{
+    /// <summary>
+    /// Window class for basic application type
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ElmWindow : IWindow
```

Comment:
Did you need create a new window class?
Why not use ElmSharp.Window?

Reply:
We need a common interface for handling window information.
It can be ElmSharp.Window, Dali window and etc..
ComponentBased application base provides a common interface for handling window information to remove EFL dependency.

Reply:
Hm. but, ElmWindow has EFL dependency
I also agree create IWindow interface to remove UI F/W dependency. 
but, `ElmWindow` could be sub class of ElmSharp.Window
``` c# 
 public class ElmWindow : ElmSharp.Window, IWindow {
```

Reply:
Yes, but we only need resource ID of the window. Therefore, I changed it into ElmWindowInfo.
I think we don't need to inherit ElmSharp.Window.